.github/workflows/icon-review.yml

```diff
@@ -36,11 +36,14 @@ jobs:
           persist-credentials: false
 
       - name: Review SVG files 🔍
+        working-directory: main
         run: |
-          cd main
-          svgFiles=$(git diff --no-index ../main ../fork --diff-filter=ACMRTUX --name-only | grep '.svg$')
-          echo "SVG files changed: ${svgFiles}"
-          npx svg-icon-review@1.1.0 --bigIcon ${svgFiles}
+          files=$(git diff --no-index ../main ../fork --diff-filter=ACMRTUX --name-only | grep '.svg$')
+          filesCount=$(echo "$files" | wc -l)
+
+          npx svg-icon-review@1.1.0 --bigIcon ${files}
+
+          echo svg_files_count=$filesCount >> $GITHUB_ENV
 
       - name: Install jq ⚙️
         run: sudo apt-get install jq
```

Comment:
Oh, I removed this local, but @okineadev beat me to push 😄
jq 1.6 comes with the runners, and without a (long, slow, and unnecessary) `apt-get update`, this is just reinstalling it. If you needed jq 1.7, you might want to do that, but jq 1.6 is fine for this (and 1.7 is coming with the 24.04 runners)

Reply:
Thanks, I didn't know that 😊 I've removed the installation of jq with this commit https://github.com/material-extensions/vscode-material-icon-theme/commit/2c27245e12f83241634daa2e7bee8dbc3e7257c9.